_vale/config/vocabularies/Docker/accept.txt

```diff
@@ -71,30 +73,35 @@ Nuxeo
 OAuth
 OTel
 Okta
+Paketo
 PKG
 Postgres
 PowerShell
 Python
 S3
+Seccomp
 SQLite
 Slack
 Snyk
 Solr
 SonarQube
 Syft
+Sysbox
 Sysdig
 Testcontainers
 Traefik
 Ubuntu
 Unix
 VMware
+VM's
```

Comment:
We should probably do this instead, to never flag possessive form of acronyms (e.g., SDK's, API's

```diff
-(?-i)[A-Z]{2,}s
+(?-i)[A-Z]{2,}'?s
```